OxfordBodleian/Juel-Jensen/BDLaethf24.xml

```diff
@@ -734,10 +740,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <history>
 
                         <origin>
-                            <origDate notBefore="1800" notAfter="1949" evidence="lettering"/>
-                            <ref target="#p3_i2"/> mentions <roleName type="title">liqa ṗāṗāsǝna ʾAbbā</roleName> Mārqos and
-                            <roleName type="title">ṗāṗāsǝna ʾAbbā</roleName> Yoḥannǝs in a supplication (<locus target="#39r"/>). These might refer to
-                            <persName ref="PRS13830MarkVII"/> and <persName ref="PRS10393Yohanne"/>, simultaneously active in <date>1747–1761</date>.
+                            <origDate notBefore="1800" notAfter="1949" evidence="lettering">nineteenth–mid-twentieth century</origDate>
+                            Unit 3 can be dated palaeographically to the nineteenth–mid-twentieth century.
+                            The <hi rendition="simple:italic"><ref target="#p3_i2">Fǝtḥat za-wald</ref></hi> mentions <hi rendition="simple:italic"><roleName type="title">liqa ṗāṗāsǝna ʾAbbā</roleName></hi> Mārqos and
```

Comment:
Perhaps a silly question: are here liqa ṗāṗāsǝna and ṗāṗāsǝna to be written with initial upper case?